### PR TITLE
Websockets: Name registration is now prepended with 'name '

### PR DESCRIPTION
**Pull request by NotRyan**
Probably works :^)

Also adds ping handling client side during name selection, since that wasn't implemented for some reason.

### CODE REVIEW

File: priv/static/site.js [hunk 1]
```diff
@@ -241,7 +243,7 @@ function checkIsAlive() {
 }
 
 function submitName(nameToSubmit) {
-	name = nameToSubmit;
+	name = "name ".concat(nameToSubmit);
```
Comment (PotatoParser) [inline priv/static/site.js]:
```javascript
name = `name ${nameToSubmit}`;
```

**Review by PotatoParser — CHANGES_REQUESTED** [review]
Nice work!

**Review by PotatoParser — APPROVED** [review]

File: server.go [hunk 2]
```diff
@@ -57,14 +57,24 @@ func socketinit(w http.ResponseWriter, r *http.Request) {
...
-			c.WriteMessage(mt, []byte("ok"))
-			defer socketHandle(c, message)
-			return
+		
+		switch messageTokens := strings.SplitAfterN(string(message), " ", 2); messageTokens[0] {
+		case "ping":
+			c.WriteMessage(mt, []byte("pong"))
+			continue
+		case "name":
+			if len(messageTokens = 2) && validateName(messageTokens[1]) {
```
Comment (BernardZhao) [inline server.go]: This is completely wrong. Fix plz

**Review by BernardZhao — CHANGES_REQUESTED** [review]

File: server.go (hunk 2 above)
Comment (NotRyan) [reply server.go]: Hey how about we cool it with the hostility on github ok?

Comment (BernardZhao) [reply server.go]: poggie

**Review by BernardZhao — APPROVED** [review]
Great. Should fix #4 unless we find more registration issues.

**Pull request merged**